### PR TITLE
Add Aleksey Tsalolikhin to Contributors list in AUTHORS

### PR DESCRIPTION
**Pull request by atsaloli**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @olehermanse can review this?

File: AUTHORS [hunk 1]
```diff
@@ -63,6 +63,7 @@ Laurent Raufaste
 Jean Remond
 Andrew Stribblehill
 James Thompson
+Aleksey Tsalolikhin
```
Comment (vpodzime) [inline AUTHORS]: Are you sure you have less than 10 commits?

**Comment by olehermanse** [comment]
@atsaloli You have made a few commits using e-mail `at433m@***`, but haven't associated this with your GitHub account. If you add this e-mail to your profile, your contributions will show up appropriately here: https://github.com/cfengine/core/graphs/contributors

File: AUTHORS (hunk 1 above)
Comment (atsaloli) [reply AUTHORS]: @vpodzime I've only made one commit to the C code  :)

@olehermanse thanks, I've added a few email accounts -- I'll verify them later today. I appreciate the tip!  :)

**Comment by atsaloli** [comment]
❤️

**Pull request merged**